pkg/install/cleanup_unix.go

```diff
@@ -209,5 +210,32 @@ func (c *CleanUpConfig) RemoveAllDirectories() error {
 	if len(msg) > 0 {
 		return fmt.Errorf("%v", strings.Join(msg, ", "))
 	}
+	removeCNILeftovers()
 	return nil
 }
+
+// remove calico and kuberouter leftover configuration files
+func removeCNILeftovers() {
+	var msg []string
+
+	calico10Conflist := "/etc/cni/net.d/10-calico.conflist"
+	calicoKubeconfig := "/etc/cni/net.d/calico-kubeconfig"
+	kuberouterConf := "/etc/cni/net.d/10-kuberouter.conflist"
+
+	if err := os.Remove(calico10Conflist); err != nil {
+		fmtError := fmt.Errorf("failed to delete %v. err: %v", calico10Conflist, err)
+		msg = append(msg, fmtError.Error())
+	}
+	if err := os.Remove(calicoKubeconfig); err != nil {
+		fmtError := fmt.Errorf("failed to delete %v. err: %v", calicoKubeconfig, err)
+		msg = append(msg, fmtError.Error())
```

Comment:
If we only use the error for logging, why wrap it first as `error` and then use only the message from it?

Reply:
replaced it with `fmt.Sprintf`